src/MSBuild.Abstractions/MSBuildHelpers.cs

```diff
@@ -142,12 +142,10 @@ public static bool ConditionToDimensionValues(string condition, out ImmutableDic
         /// </summary>
         public static bool FrameworkHasAValueTuple(string tfm)
         {
-            return tfm.ContainsIgnoreCase(MSBuildFacts.NetstandardPrelude)
-                || tfm.ContainsIgnoreCase(MSBuildFacts.NetcoreappPrelude)
-                ? false
-                : !tfm.StartsWith("net", StringComparison.OrdinalIgnoreCase)
-                    ? false
-                    : tfm.StartsWith(MSBuildFacts.LowestFrameworkVersionWithSystemValueTuple);
+            return !tfm.ContainsIgnoreCase(MSBuildFacts.NetstandardPrelude)
+                && !tfm.ContainsIgnoreCase(MSBuildFacts.NetcoreappPrelude)
+                && (tfm.StartsWith("net", StringComparison.OrdinalIgnoreCase)
+                    && tfm.StartsWith(MSBuildFacts.LowestFrameworkVersionWithSystemValueTuple));
```

Comment:
Roslyn suggested that I do this and so I did

Reply:
Hope I never need to debug this :(

---
In reply to: [419817573](https://github.com/dotnet/try-convert/pull/285#discussion_r419817573) [](ancestors = 419817573)